mongodbatlas/data_source_mongodbatlas_cluster_outage_simulation.go

```diff
@@ -0,0 +1,84 @@
+package mongodbatlas
```

Comment:
Please add _test.go files before merging

Reply:
yes, _test.go files will be added, we are trying to figure out ways to reduce the size of ours PRs when adding a new resource to TF which tend to be quite large. Hence, we're breaking the feature out.

Reply:
ah makes sense! thanks for sharing and resolving